helm/examples/azure/log-analytics/loganalytics-hpcc-logaccess.yaml

```diff
@@ -27,6 +27,7 @@ global:
       searchColumn: "Name"
       keyColumn: "ContainerID"
       timeStampColumn: "TimeGenerated"
+      disableJoins: false #Potentially expensive join operations needed to fetch a certain column can be disabled
```

Comment:
Considered adding a higher level config option to disable any/all expensive operations, but this feels like the adequate level.